### DISCUSSION TITLE
How do you create an updated-on field in MariaDB

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#8876] [Help] [answered]
Hi Guys,

How do I create an updated-on field in MariaDB that works with BudiBase? I have tried using the `ON UPDATE CURRENT_TIMESTAMP()` method within MariaDB, but that doesn't update when I change a record through the published BudiBase app. It works fine if I update through phpmyadmin though.

**Comment by melohagan** [comment, marked as answer]
Hey @FlaminWrap 

The `ON UPDATE` constraint isn't automatically supported via the **Save Row** action, however you can write a custom query.

![Screenshot 2022-12-06 at 08 38 47](https://user-images.githubusercontent.com/101575380/205861729-bbb7b3c9-ac89-45ac-bde6-0ca22002031b.png)

Timestamp is automatically updated:
![Screenshot 2022-12-06 at 08 40 06](https://user-images.githubusercontent.com/101575380/205862006-43fab967-8821-477d-9bd8-0bac373d31af.png)
